qemu/tests/cfg/arm_cpu_test_clusters.cfg

```diff
@@ -0,0 +1,18 @@
+- arm_cpu_test_clusters:
+    type = arm_cpu_test_clusters
+    kill_vm_on_error = yes
+    start_vm = no
+    only aarch64
+    # support on RHEL.9.1
+    no RHEL.8 RHEL.9.0
+    required_qemu = [7.0.0, )
+    vcpu_maxcpus = 0
```

Comment:
make smp=vcpu_maxcpus=0 safer, to avoid users defining SMP by themselves. Other LGTM.

Reply:
good suggestion

Reply:
corrected